### ISSUE TITLE
Paper trading dialog doesn't close after TWS is launched.

### ISSUE THREAD

**Issue by xerges-crypto** [IbcAlpha/IBC#101]
I just installed a IBC on my Linux machine but is not working as expected because IBC doesn't close the "Warning" dialog that appears on paper trading accounts despite It seems that can detect the dialog (as shown in log files).

Can you give me what I am missing please? (On windows works fine)
Thanks in advance

**Version**
IBC 3.8.4-beta.2

**Sequence**
IBC starts normally , TWS is launched, Warning dialog "This is not a brokerage account..." appears, but IBC don't close it.

Testing a different port to see any different behavior doesn't change anything, IBC open settings, change port and close config dialog, but the Warning dialog persists until I press the "I understand button".

**Environment:**

Ubuntu 20.04
 java --version 
openjdk 14.0.2 2020-07-14
OpenJDK Runtime Environment (build 14.0.2+12-Ubuntu-120.04)
OpenJDK 64-Bit Server VM (build 14.0.2+12-Ubuntu-120.04, mixed mode, sharing)

**log**
...
020-11-21 19:55:29:873 IBC: Creating main window future
2020-11-21 19:55:30:844 IBC: Detected frame entitled: Login; event=Opened
2020-11-21 19:55:30:902 IBC: Detected frame entitled: Login; event=Activated
2020-11-21 19:55:30:903 IBC: Setting Trading mode = paper
2020-11-21 19:55:30:977 IBC: Setting user name
2020-11-21 19:55:30:979 IBC: Setting password
2020-11-21 19:55:30:982 IBC: Detected frame entitled: Login; event=Focused
2020-11-21 19:55:31:096 IBC: Click button: Paper Log In
2020-11-21 19:55:31:242 IBC: Button now disabled: Paper Log In
2020-11-21 19:55:31:711 IBC: Detected frame entitled: Login; event=Lost focus
2020-11-21 19:55:31:717 IBC: Detected frame entitled: Authenticating...; event=Opened
2020-11-21 19:55:31:825 IBC: Detected frame entitled: Authenticating...; event=Activated
2020-11-21 19:55:31:826 IBC: Detected frame entitled: Authenticating...; event=Focused
2020-11-21 19:55:34:158 IBC: Detected frame entitled: Retrieving settings key...; event=Lost focus
2020-11-21 19:55:34:282 IBC: Detected dialog entitled: Downloading settings from server; event=Closed
LogModuleConfigurator-Init: Log4j Ver2.x found on classpath
LogModuleConfigurator-Init: LogModuleConfigurator initialized with Log4j Verd.x
2020-11-21 19:55:46:322 IBC: Detected frame entitled: DUxxxxx Interactive Brokers (Simulated Trading); event=Opened
2020-11-21 19:55:46:322 IBC: Found TWS main window
2020-11-21 19:55:46:323 IBC: Login has completed
2020-11-21 19:55:46:323 IBC: Got main window from future
2020-11-21 19:55:46:389 IBC: Detected frame entitled: DUxxxxx Interactive Brokers (Simulated Trading); event=Activated
2020-11-21 19:55:46:395 IBC: Detected frame entitled: DUxxxxx Interactive Brokers (Simulated Trading); event=Focused
2020-11-21 19:55:46:483 IBC: Detected frame entitled: Starting application...; event=Closed
2020-11-21 19:55:48:923 IBC: **Detected dialog entitled: Warning; event=Opened**
2020-11-21 19:55:49:108 IBC: Detected frame entitled: DUxxxxx Interactive Brokers (Simulated Trading); event=Lost focus
2020-11-21 19:55:49:109 IBC: **Detected dialog entitled: Warning; event=Activated**
2020-11-21 19:55:49:110 IBC: **Detected dialog entitled: Warning; event=Focused**
2020-11-21 19:55:49:558 IBC: Detected dialog entitled: DUxxxxx Trader Workstation Configuration (Simulated Trading); event=Opened
2020-11-21 19:55:49:558 IBC: Got config dialog from future
2020-11-21 19:55:49:792 IBC: Performing port configuration
2020-11-21 19:55:49:920 IBC: TWS API socket port is already set to 9999
2020-11-21 19:55:49:921 IBC: Click button: OK
2020-11-21 19:55:50:321 IBC: Detected dialog entitled: DUxxxxx Trader Workstation Configuration (Simulated Trading); event=Closed

*(now I manually close the Warrning Dialog)*

2020-11-21 19:55:59:223 IBC: Detected dialog entitled: Warning; event=Lost focus
2020-11-21 19:56:50:592 IBC: Detected dialog entitled: Warning; event=Activated
2020-11-21 19:56:50:595 IBC: Detected dialog entitled: Warning; event=Focused
2020-11-21 19:56:54:782 IBC: Detected dialog entitled: Warning; event=Lost focus

**Windows Machine**

Windows machine is using JRE 1.8

C:\Users\user>C:\Users\user\.i4j_jres\1.8.0_152-tzdata2019c_64\bin\java
sion
java version "1.8.0_152"
Java(TM) SE Runtime Environment (build 1.8.0_152-b16)
Java HotSpot(TM) 64-Bit Server VM (build 25.152-b16, mixed mode)

C:\Users\user>

Comparing with the log on my Windows (successful) case it seems like IBC doesn't receive an *Opened* event to Click the button.

I hope this helps

2020-11-21 20:18:32:998 IBC: Found TWS main window
2020-11-21 20:18:32:998 IBC: Login has completed
2020-11-21 20:18:33:007 IBC: Got main window from future
2020-11-21 20:18:33:660 IBC: Detected frame entitled: Starting application...; event=Closed
2020-11-21 20:18:37:672 IBC: Detected frame entitled: DUxxxxx Interactive Brokers (Simulated Trading); event=Activated
2020-11-21 20:18:37:674 IBC: Detected frame entitled: DUxxxxx Interactive Brokers (Simulated Trading); event=Focused
2020-11-21 20:18:37:675 IBC: Detected frame entitled: DUxxxxx Interactive Brokers (Simulated Trading); event=Lost focus
2020-11-21 20:18:37:675 IBC: Detected dialog entitled: Warning; event=Activated
2020-11-21 20:18:37:677 IBC: Detected dialog entitled: Warning; event=Focused
2020-11-21 20:18:37:677 IBC: **Detected dialog entitled: Warning; event=Opened**
2020-11-21 20:18:37:678 IBC: **Click button: I understand and accept**
2020-11-21 20:18:38:834 IBC: Detected dialog entitled: Warning; event=Lost focus
2020-11-21 20:18:38:834 IBC: Detected frame entitled: DUxxxxx Interactive Brokers (Simulated Trading); event=Activated
2020-11-21 20:18:38:840 IBC: Detected frame entitled: DUxxxxx Interactive Brokers (Simulated Trading); event=Focused
2020-11-21 20:18:38:851 IBC: Detected dialog entitled: Warning; event=Closed

**Going back to Ubuntu**

I've tried to use JRE 1.8 in Ubuntu, but the dialog still remains on the screen.

Thanks for any help.

**Comment by xerges-crypto** [comment]
Note: same environment but using **iBC 3.8.2 works fine**.

**Comment by rlktradewright** [comment]
You probably don't have:

`AcceptNonBrokerageAccountWarning=yes`

in your config.ini.

Please, when you want to show information from the logfile, just attach it to your post (ie don't quote it inline) and include the whole log, not just the parts that you think might be relevant. In this case, the most relevant part is in what you left out, ie the list of settings in config.ini. Without that I can't tell whether my supposition is correct (though it's 99.9999% likely to be correct).

**Comment by xerges-crypto** [comment]
Thanks. 
Related to the logs you're right. The IBC settings are shown at the beginning of the log file. As I can't see a way to attach the whole file, I was trying to help 'reducing' the noise. Clearly I was wrong. 

I restart the Ubuntu and everything works fine, even compiling from the sources, so clearly its was a local issue with my environment.

Finally I was pretty sure that I used the default config.ini settings (where this option is set by default). As I was getting crazy with the issue inspect the changes from 3.8.2 and there was nothing related with the issue, so maybe yesterday I was really tired and somehow I mixed o remove this seting ;)

Now everything is working fine.

Thanks for your time.
